lawlibrary/templates/lawlibrary/article_detail.html

```diff
@@ -0,0 +1,19 @@
+<h1>Article Detail</h1>
```

Comment:
These top-level templates can extend the basic peachjam template (main.html I think?) so that they get the basic layout.